dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+pytest
```

Comment:
What we chose to do in Python is to ping the version at release time but not in master.
See: https://github.com/Kinto/kinto/blob/master/setup.py#L18-L21 vs https://github.com/Kinto/kinto/blob/1.3.1/setup.py#L18-L21